lib/SampleService/core/config.py

```diff
@@ -207,15 +231,46 @@ def get_validators(url: str) -> MetadataValidatorSet:
     :returns: A set of metadata validators.
     '''
     # TODO VALIDATOR make validator CLI
+
     try:
-        with _urllib.request.urlopen(url) as res:
-            cfg = _yaml.safe_load(res)
+        config_asset = None
+        if url:
+            config_url = url
+        elif not repo_path:
+            raise ValueError(f'No metadata validator config URL or repo path.')
+        else:
+            try: 
+                repo = _Github(login_or_token=token).get_repo(repo_path)
+                releases = [rel for rel in repo.get_releases() if prerelease_ok or not rel.prerelease]
+            except:
+                raise RuntimeError(f'Fetching releases from repo {repo_path} failed.')
```

Comment:
```suggestion
            except Exception as e:
                raise RuntimeError(f'Fetching releases from repo {repo_path} failed.') from e
```

Otherwise debugging a fail is going to suck. I have seen this. I have done this. You do not want this